code/game/objects/items/devices/scanners.dm

```diff
@@ -407,11 +407,20 @@ GENE SCANNER
 	if(istype(M) && M.reagents)
 		var/render_list = list()
 		if(M.reagents.reagent_list.len)
-			render_list += "<span class='notice ml-1'>Subject contains the following reagents:</span>\n"
+			render_list += "<span class='notice ml-1'>Subject contains the following reagents in their blood:</span>\n"
 			for(var/datum/reagent/R in M.reagents.reagent_list)
 				render_list += "<span class='notice ml-2'>[round(R.volume, 0.001)] units of [R.name][R.overdosed ? "</span> - <span class='boldannounce'>OVERDOSING</span>" : ".</span>"]\n"
 		else
-			render_list += "<span class='notice ml-1'>Subject contains no reagents.</span>\n"
+			render_list += "<span class='notice ml-1'>Subject contains no reagents in their blood.</span>\n"
+		var/obj/item/organ/stomach/belly = M.getorganslot(ORGAN_SLOT_STOMACH)
+		if(belly)
+			if(belly?.reagents.reagent_list.len)
```

Comment:
The existence of `belly` was already checked for in the previous line.

Reply:
Will fix it